client/settings/phone-input/iti.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'iti/utils';
```

Comment:
These files are usually named `declarations.d.ts` in the code base. Maybe you'll want to stick to that naming?